pyspecProcScripts/correlation_alignment.py

```diff
@@ -219,12 +236,13 @@ def correl_align(
             psd.strm(
                 "signal energy per transient (recalc to check that it stays"
                 " the same):",
-                (abs(s_copy**2).data.sum().item() / N),
+                (abs(for_Ecalc**2).data.sum().item() / N),
             )
         )
-
+        # {{{ Calculate energy difference from last shift to see if
+        #     there is any further gain to keep reiterating
```

Comment:
again I think this is what we are doing so that we don't need to keep reiterating if we don't need to

Reply:
yes!